checker/jdk/nullness/src/java/lang/Object.java

```diff
@@ -10,7 +10,7 @@
   public Object() { throw new RuntimeException("skeleton method"); }
   @Pure public boolean equals(@Nullable Object a1) { throw new RuntimeException("skeleton method"); }
   // toString() is @SideEffectFree rather than @Pure because it returns a string
-  // that differs according to ==, and @Deterministic is with respect to ==.
+  // that differs according to ==.  It would be @Deterministic is with respect to ==.
```

Comment:
I don't know why there are changes to this - it's unrelated and probably an artifact of this PR being fairly old. I'll remove this change.